pkg/math/oracle/median.go

```diff
@@ -32,9 +33,9 @@ func NewMedianAggregator(logger *zap.Logger, cfg mmtypes.MarketMap) (*MedianAggr
 	}
 
 	return &MedianAggregator{
-		logger:          logger,
-		cfg:             cfg,
-		PriceAggregator: types.NewPriceAggregator(),
+		logger:         logger,
+		cfg:            cfg,
+		DataAggregator: aggregator.NewDataAggregator[string, types.TickerPrices](),
```

Comment:
are we removing the price aggregator type alias?

Isn't it still just 

```go
type PriceAggregator = DataAggregator[string, TickerPrices]
```

Reply:
i had to remove it since the oracle expects a `DataAggregator` type directly but we inherit it so now we just use the interface instead.